app/src/main/java/com/eveningoutpost/dexdrip/Home.java

```diff
@@ -125,6 +142,10 @@ public void onReceive(Context ctx, Intent intent) {
         updateCurrentBgInfo();
     }
 
+    public static Context getContext() {
```

Comment:
the service that calls into a class (or the activity) would pass its own context along, it seems the only place you are using this is in the BgSendQueue class which is handed a context whenever it is used already, we should just use that same context